scripts/install.sh

```diff
@@ -7,8 +7,15 @@
 
 set -u
 
+BINARY_DOWNLOAD_PREFIX="https://github.com/apollographql/router/releases/download"
+
+# Router version defined in apollo-router's Cargo.toml
+# Note: Change this line manually during the release steps.
+PACKAGE_VERSION="v0.16.0"
```

Comment:
NIT: DEFAULT_VERSION ?

Reply:
One of the key components of this design is that there is no such thing as a "default" version as the script only installs the version of the artifact it was designed for.

Maybe you're just bike-shedding the `PACKAGE` part?

Reply:
I am...

Reply:
Ok. Since it's identical to Rover now and it's not a user-configurable variable, I'm going to leave it as is.